java/src/main/java/com/logicalclocks/base/engine/FeatureGroupBaseEngine.java

```diff
@@ -1,33 +1,31 @@
 /*
- * Copyright (c) 2020 Logical Clocks AB *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ *  Copyright (c) 2020-2022. Hopsworks AB
  *
- * http://www.apache.org/licenses/LICENSE-2.0
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ *  See the License for the specific language governing permissions and limitations under the License.
  *
- * See the License for the specific language governing permissions and limitations under the License.
  */
 
-package com.logicalclocks.hsfs.engine;
+package com.logicalclocks.base.engine;
 
-import com.logicalclocks.hsfs.EntityEndpointType;
-import com.logicalclocks.hsfs.Feature;
-import com.logicalclocks.hsfs.FeatureGroup;
-import com.logicalclocks.hsfs.FeatureStoreException;
-import com.logicalclocks.hsfs.ExternalFeatureGroup;
-import com.logicalclocks.hsfs.StreamFeatureGroup;
-import com.logicalclocks.hsfs.metadata.FeatureGroupApi;
-import com.logicalclocks.hsfs.metadata.FeatureGroupBase;
-import com.logicalclocks.hsfs.metadata.TagsApi;
+import com.logicalclocks.base.metadata.FeatureGroupApi;
+import com.logicalclocks.base.metadata.FeatureGroupBase;
+import com.logicalclocks.base.metadata.TagsApi;
+import com.logicalclocks.base.EntityEndpointType;
+import com.logicalclocks.base.Feature;
+import com.logicalclocks.base.FeatureStoreException;
 
 import java.io.IOException;
 import java.util.ArrayList;
-import java.text.ParseException;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
```

Comment:
also maybe the class name should be changed, otherwise, it is inconsistent with other classes with this name. Usually base class is extended by a child class, but in this case, `FeatureGroupBaseEngine` has nothing to do with `FeatureGroupEngine` (functionality wise)

Reply:
cant we make `FeatureGroupEngine` inherit `FeatureGroupBaseEngine` and resolve it that way?